### PR TITLE
make banner dynamic

### PR DESCRIPTION
**Pull request by gabimartini**
Screenshot 2022-06-20 at 15 21 32
Screenshot 2022-06-20 at 15 21 52

Trello issue: https://trello.com/c/TUKHfabe
https://trello.com/c/0teya5vW
https://trello.com/c/nCYG9oJ4

### CODE REVIEW

**Review by em-cd — APPROVED** [review]
Very nice, it looks great! 🎉 

Before you merge it, can you please deploy it to staging and ask Alex to test it? Then if he's happy with it go ahead and merge! 🚢

**Comment by gabimartini** [comment]
> Very nice, it looks great! 🎉
> 
> Before you merge it, can you please deploy it to staging and ask Alex to test it? Then if he's happy with it go ahead and merge! 🚢

Yes! Should I update staging with all the branch that we approved?

**Comment by em-cd** [comment]
Yes, you can do, that way we can get Alex to QA all of them at once :)

File: app/models/book_adoption_eve_page.rb [hunk 1]
```diff
@@ -6,6 +6,9 @@ class BookAdoptionEvePage < Fae::StaticPage
   def self.fae_fields
     {
       hero_image: { type: Fae::Image },
+      banner_tag: { type: Fae::TextField },
+      lede_text: { type: Fae::TextField },
```
Comment (em-cd) [inline app/models/book_adoption_eve_page.rb:10]: Do `banner_tag` and `lede_text` need to be removed from here now?

**Review by em-cd — COMMENTED** [review]
Looks good! I just had one comment inline, but then this is good to go! 👍

File: app/models/book_adoption_eve_page.rb (hunk 1 above)
Comment (gabimartini) [reply app/models/book_adoption_eve_page.rb]: I removed and added banner in the plan page and event page

**Review by em-cd — APPROVED** [review]
Nice work!

**Pull request merged**